internal/service/globalclusterconfig/resource_global_cluster_config_test.go

```diff
@@ -273,41 +273,65 @@ const (
 		replication_specs {
 			zone_name  = "US"
 			num_shards = 1
-			regions_config {
+			region_configs {
+				auto_scaling {
+					disk_gb_enabled = false
+				}
 				region_name     = "US_EAST_1"
-				electable_nodes = 3
+				provider_name	= "AWS"
```

Comment:
are you planning to move rep specs to testutil packages in following PRs?

Reply:
Yes